src/parser/av1_parser.cpp

```diff
@@ -112,10 +114,327 @@ ParserResult Av1VideoParser::ParsePictureData(const uint8_t *p_stream, uint32_t
             default:
                 break;
         }
+
+        // Init Roc decoder for the first time or reconfigure the existing decoder
+        if (new_seq_activated_) {
+            if (NotifyNewSequence(&seq_header_, &frame_header_) != PARSER_OK) {
+                return PARSER_FAIL;
+            }
+            new_seq_activated_ = false;
+        }
+
+        // Submit decode when we have the entire frame data
+        if (tile_group_data_.tile_number && tile_group_data_.tg_end == tile_group_data_.num_tiles - 1) {
+            // Temp code for intra testing
+            /* if (FindFreeInDecBufPool() != PARSER_OK) {
```

Comment:
Is this code going to be used? Or else please remove

Reply:
Yes in the following PR.